arch/loongarch/compare256_lasx.c

```
@@ -7,9 +7,8 @@
 #include "zbuild.h"
```

Cppcheck (2.19.0)

[information] Include file

(missingInclude)